docs/setup.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[setup]]
+== Set up the Agent
+
+:source-highlighter: coderay
+
+[discrete]
+[[requirements]]
+=== Requirements
+
+This project requires Swift `5.3`, and is intended for use in Swift-base mobile apps.
+
+[discrete]
+[[add-agent-dependency]]
+=== Add the Agent dependency
+Add the Elastic APM iOS Agent to your Xcode project or your `Package.swift`.
+
+Here are instructions for adding a https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app[package dependency] to a standard Xcode poject.
+
+Details of adding dependencies to your Package.swift can be found on https://developer.apple.com/documentation/xcode/creating_a_standalone_swift_package_with_xcode#3578941['Add a Dependency on Another Swift Package'].
+Below is a helpful code-snippet:
+
+`package.swift`:
+[source,swift,linenums,highlight=2;10]
```

Comment:
I've never seen this before. What do items 2 and 3 do?

Reply:
swift & linenums?
I was trying to tag the code highlighting style to the swift language. 
linenums add line numbers to the code snippet which is apparently a requirement to get highlighting to those lines on 2 & 10.
found the details here : https://docs.asciidoctor.org/asciidoc/latest/verbatim/highlight-lines/

Reply:
Sorry, I was using zero-based numbering -- but you've still answered my question. TIL!